### DISCUSSION TITLE
Direct open a Color corrections filter

### DISCUSSION THREAD

**Discussion by tomsoel** [viliusle/miniPaint#302] [General]
Hello,
Thank you for this awesome tool!
Sorry if my question may seem obvious. I use minipaint on my site to allow users to change the hue and saturation of images that I offer for download, using the url mywebsite/minipaint/?image=...
Is it possible to use an url link that would directly open the "Color Corrections" dialog box?
Thx a lot!

**Comment by viliusle** [comment]
Not yet, but required code is ready (try F3). So it would be easy to implement it. 
You should create issue about it and mention to support open external image automatically AND open some related action.

**Reply by tomsoel** [reply]
Thanks for reply. 
I'm going to look at this ;)

**Reply by tomsoel** [reply]
Hello,
Just another question. Where are stored images opened by users ?
Is there a folder that needs to be cleaned to avoid cluttering the server?
Thx a lot!

**Reply by viliusle** [reply]
Images are stored on browser memory, there are no server, and images are not uploaded anywhere, at least in official version.

**Reply by tomsoel** [reply]
Thanks for reply! Nice!
So my website host wont be solicited? Awesome ;)

**Reply by viliusle** [reply]
If you want to have custom version, you will still need to host source files on server, but that's all. Server will use minimal resources for that, only to provide files on load.
